docs/content/deployment/dagster-daemon.mdx

```diff
@@ -1,36 +1,119 @@
 ---
-title: Dagster Daemon | Dagster
+title: Dagster daemon | Dagster
 description: Several Dagster features require a long-running daemon process within your deployment.
 ---
 
-# Dagster Daemon
+# Dagster daemon
 
 Several Dagster features, like [schedules](/concepts/partitions-schedules-sensors/schedules), [sensors](/concepts/partitions-schedules-sensors/sensors), and [run queueing](/deployment/run-coordinator#limiting-run-concurrency), require a long-running `dagster-daemon` process to be included with your deployment.
 
-To start the `dagster-daemon` process locally, launch the following command in the same folder as your [`workspace.yaml`](/concepts/code-locations/workspace-files#understanding-workspace-files) file and keep the process running:
+---
+
+## Starting the daemon
+
+To start the `dagster-daemon` process locally, launch the following command in the same folder as your [workspace file (`workspace.yaml`)](/concepts/code-locations/workspace-files) file and keep the process running:
```

Comment:
Daemon does not require a `workspace.yaml` file. It will discover `pyproject.toml` and also supports `-m` and `-f`

Reply:
You're totally right - thanks for the reminder. Addressed.

Reply:
I addressed this by adapting [this section of the Code location docs](https://docs.dagster.io/concepts/code-locations#local-development).